src/controllers/user.controller.js

```diff
@@ -267,4 +271,39 @@ export class UserController {
       });
     }
   }
+
+  static async forgotpass(req, res) {
+    try {
+    const user = req.user
+    console.log(user)
+        const resetLink = JWT.sign({ user_email: user.email,
+         user_id: user.id}, 
+          process.env.SECRET_TOKEN, { expiresIn: '10m' });
+          const link = `${process.env.BASE_URL}/reset-password?token=${resetLink}`
+          const resetMessage = resetPasswordTemplate(user.email,link)
+          sendEmail( 
+            emailConfig({
+            email: user.email,
+            subject: "reset password",
+            content: resetMessage,
+          })
+          );
+        return res.status(200).json({ message: link} );
+      } catch(error) {
+      res.status(500).json({ error: error.message });
+    }
+    }
+    
+    static async resetpass(req, res) {
+      try {
+      const id = req.id
+      const { newPassword } = req.body
+        const password = await BcryptUtility.hashPassword(newPassword)
+        await UserService.updateUser({ password },id)
+        return res.status(200).json({ message:"password updated"})
+      } catch (error) {
+        return res.status(500).json({error: "not reset"})
+      }
```

Comment:
- Rename your controller to follow camelCase convention
- Update the successful message
- The catch block should follow the same convention implemented on the registered user. Ask Yves